ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -333,43 +333,39 @@
       ^{:key label}
       [labeled-icon (stylefy/use-style style/transit-changes-legend-icon) [icon] label]))])
 
-(defn format-range [{:keys [lower upper lower-inclusive? upper-inclusive?]}]
+
+(defn format-range [lower upper]
```

Comment:
Is this function meant to handle a lower=nil upper=non-nil case (format-range nil 42) produces "—2"

Reply:
True, was wonderingthe same during implementation but apparently even old code didn't handle that scenario. Logic on backend side sets lower boundary to same as upper if it's missing.
Nevertheless I'll consider improving that in the future.